src/nanoemoji/write_font.py

```diff
@@ -229,51 +236,114 @@ def _next_name(ufo: ufoLib2.Font, name_fn) -> str:
     return name_fn(i)
 
 
-def _create_glyph(color_glyph: ColorGlyph, painted_layer: PaintedLayer) -> Glyph:
+class ReuseResult(NamedTuple):
+    glyph_name: str
+    transform: Affine2D
+
+
+class GlyphReuseCache:
```

Comment:
@anthrotype notes that there may be a threshold where cost of reuse is higher than cost of emitting the glyph (think triangle). Maybe don't fix here but file an issue to investigate?

Reply:
Tracking in #302